addons/project_timesheet_holidays/models/project_task.py

```diff
@@ -11,7 +11,7 @@ class ProjectTask(models.Model):
     is_timeoff_task = fields.Boolean("Is Time off Task", compute="_compute_is_timeoff_task", search="_search_is_timeoff_task", export_string_translation=False)
 
     def _compute_leave_types_count(self):
-        timesheet_read_group = self.env['account.analytic.line']._read_group(
+        timesheet_read_group = self.env['account.analytic.line'].sudo()._read_group(
```

Comment:
Currently, the is_timeoff_task field is being loaded even when no access to the timesheet module exists. However, this field is intended to be used within the timesheet module, causing the error. We need to ensure that the is_timeoff_task field is only loaded when the user has access to the timesheet.